samples/test-samples.cmake

```diff
@@ -20,7 +20,7 @@ execute_process(COMMAND ${CMAKE_COMMAND} -E env DESTDIR=${BUILD_DIR}/install ${C
 set(INSTALL_DIR ${BUILD_DIR}/install${PREFIX_DIR})
 
 # TODO: Add the rest of the samples.
```

Comment:
Remove this TODO if we've added the rest of the samples 😉

Reply:
:)